pcalcl.l

```diff
@@ -81,7 +81,30 @@ extern  int lineno;
                                 return(NUMBER);
                                 }
 
-[0-9]+([0-9_,]*[0-9]+)?         {
+KB                              {
```

Comment:
i was thinking this would be `[KkMmGgTtPpEeZzYy][Bb]` and they'd all be multiples of 1000

Reply:
I would prefer having [KkMmGgTtPpEeZzYy][B] standing for powers of two (this is more convenient for programmers). This is inline with JEDEC notation.

Using [KkMmGgTtPpEeZzYy][b] is tricky, that could mean kilobit.

Reply:
keeping `B` for byte and omitting `b` for bit sounds fine